experimental/adaptive-cards-2/Catering/Views/Home/IndexOld.cshtml

```diff
@@ -0,0 +1,98 @@
+﻿@model Catering.Models.ChatConfig
```

Comment:
Is this IndexOld.html a deliberate inclusion?

Reply:
No, I removed it.